themes/default/content/blog/pulumi-release-notes-66/index.md

```diff
@@ -0,0 +1,113 @@
+- [Remote provisioning of an EC2 instance]({{<relref "/registry/packages/command/#remote-provisioning-of-an-ec2-instance">}})
+- [Invoking a Lambda during a Pulumi deployment]({{<relref "/registry/packages/command/#invoking-a-lambda-during-pulumi-deployment">}})
+- [Using local.Command with curl to manage an external REST API]({{<relref "/registry/packages/command/#invoking-a-lambda-during-pulumi-deployment">}})
+- [Graceful cleanup of workloads in a Kubernetes cluster]({{<relref "/registry/packages/command/#graceful-cleanup-of-workloads-in-a-kubernetes-cluster">}})
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/99)
+
+### Support `pulumi import` for Kubernetes CRDs
+
+We have added `pulumi import` support for Kubernetes CustomResourceDefiniton (CRD). Now the spec of a CRD will be imported during `pulumi import`. The same fix improves input generation for other Kubernetes resources as well, providing significantly better fidelity in covering inputs for existing resources.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes/issues/1410)
+
+### Various improvements to Helm `Release`
+
+This milestone we spent some time making improvements to the Helm `Release` support. Of particular note are the ability to import existing Helm releases installed via the Helm command line into Pulumi (#1818 ) and the ability to supply Helm values through YAML files (#1828). In addition, we have made a variety of bug fixes this iteration to make Helm Release a more robust option to use for your Kubernetes environment.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi-kubernetes/pull/1818) and [this GitHub issue](https://github.com/pulumi/pulumi-kubernetes/pull/1828).
+
+## Pulumi CLI and core technologies
+
+### Support using native ES modules as Pulumi scripts
+
+Native ECMAScript module (ESM) support has been added for the Node.js SDK. Pulumi users can now use Pulumi in projects with "type": "module" configured.
+
+[Learn more in this GitHub issue](https://github.com/pulumi/pulumi/issues/7764)
+
+### Support packages with plugins hosted in any third-party location
+
+Pulumi Packages can now host their plugins anywhere (like GitHub releases) instead of needing to be published by Pulumi. We now detect any dependency that contains pulumi-plugin.json and treat it as a Pulumi Package, automatically downloading associated plugins as needed. To support this, the freshly generated Multi-Language Component (MLC) plugin will now include pulumi-plugin.json by default.
+
+Learn more in the following GitHub issues: [#8515](https://github.com/pulumi/pulumi/issues/8515), [#8516](https://github.com/pulumi/pulumi/issues/8516), [#8517](https://github.com/pulumi/pulumi/issues/8517), [#8530](https://github.com/pulumi/pulumi/issues/8530), [#8527](https://github.com/pulumi/pulumi/issues/8527) and [#8532](https://github.com/pulumi/pulumi/issues/8532).
```

Comment:
these ones really need some kind of link text that isn't just the issue number, can we use the issue title instead?